queries/javascript.jsx.scm

```diff
@@ -1,56 +1,82 @@
+;;!! <foo>bar</foo>
+;;!  ^^^^^^^^^^^^^^
+;;!       ###
+;;!       ***
 (
   (jsx_element) @xmlElement @_.interior @_.iteration
   (#child-range! @_.interior 0 -1 true true)
   (#child-range! @_.iteration 0 -1 true true)
 )
 
+;;!! <foo>bar</foo>
+;;!       ***
 (
   (jsx_element) @xmlStartTag.iteration @xmlEndTag.iteration @xmlBothTags.iteration
   (#child-range! @xmlStartTag.iteration 0 -1 true true)
   (#child-range! @xmlEndTag.iteration 0 -1 true true)
   (#child-range! @xmlBothTags.iteration 0 -1 true true)
 )
 
+;;!! <foo>bar</foo>
+;;!  ^^^^^---------
 (jsx_element
   (jsx_opening_element) @xmlStartTag @xmlBothTags
   (#allow-multiple! @xmlBothTags)
 ) @_.domain
 
+;;!! <foo>bar</foo>
+;;!  --------^^^^^^
 (jsx_element
   (jsx_closing_element) @xmlEndTag @xmlBothTags
   (#allow-multiple! @xmlBothTags)
 ) @_.domain
 
+;;!! <foo/>
```

Comment:
If there's no annotation below, it's implied that the whole thing is the content range and domain